c-api/ResvgQt.h

```diff
@@ -530,6 +530,7 @@ class ResvgRenderer {
      */
     QImage renderToImage(const QSize &size = QSize()) const
     {
+        resvg_transform xf = resvg_transform_default();
```

Comment:
This function should accept `QTransform` then, but I guess I can add this myself.
At least rename `xf` to `ts`.

Reply:
Please go ahead and refactor to incorporate `QTransform`.